.travis.yml

```diff
@@ -18,6 +18,7 @@ addons:
     - ubuntu-toolchain-r-test
     packages:
     - g++-4.8
+    - docker-ce
```

Comment:
This may not be necessary anymore, but seemed harmless to leave in just for explicitness.